### PR TITLE
Edit the localization file of zh-TW

### PR DESCRIPTION
**Pull request by ralphchung**
I'm a Taiwanese user and have edited the zh-TW.json based on the original version.
The use of words in this document is more in keeping with the habits of how Taiwanese use Chinese.

### CODE REVIEW

**Comment by Rokt33r** [comment]
@rayou Could you review this?

**Comment by rayou** [comment]
No problem, will do this weekend.

File: locales/zh-TW.json [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
-	"Preferences": "首選項",
-	"Make a note": "新建筆記",
+	"Preferences": "偏好設定",
+	"Make a note": "做點筆記",
 	"Ctrl": "Ctrl",
 	"Ctrl(^)": "Ctrl",
-	"to create a new note": "新建筆記",
+	"to create a new note": "新增筆記",
 	"Toggle Mode": "切換模式",
 	"Trash": "廢紙簍",
```
Comment (rayou) [inline locales/zh-TW.json]: 改用`垃圾桶`比較符合台灣用語

File: locales/zh-TW.json [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
-	"Storages": "本地儲存",
-	"Add Storage Location": "添加一個本地儲存位置",
-	"Add Folder": "新建資料夾",
-	"Open Storage folder": "打開一個本地儲存位置",
-	"Unlink": "取消鏈接",
+	"Your preferences for Boostnote": "Boostnote 偏好設定",
+	"Storages": "本機儲存空間",
+	"Add Storage Location": "新增一個本機儲存位置",
+	"Add Folder": "新增資料夾",
+	"Open Storage folder": "開啟一個本機儲存位置",
```
Comment (rayou) [inline locales/zh-TW.json]: 開啟儲存資料夾
image

File: locales/zh-TW.json [hunk 3]
```diff
@@ -1,91 +1,91 @@
...
+	"When Editor Blurred": "當編輯器失去焦點時",
+	"When Editor Blurred, Edit On Double Click": "當編輯器失去焦點時，雙擊切換到編輯畫面",
+	"On Right Click": "點擊右鍵切換兩個頁面",
 	"Editor Keymap": "編輯器 Keymap",
-	"default": "默認",
+	"default": "預設",
 	"vim": "vim",
 	"emacs": "emacs",
-	"⚠️ Please restart boostnote after you change the keymap": "⚠️ 設置好快捷鍵後，記得重啟设置好快捷键后，记得重启boostnote",
+	"⚠️ Please restart boostnote after you change the keymap": "⚠️ 請重新開啟 Boostnote 已完成設定。",
```
Comment (rayou) [inline locales/zh-TW.json]: 以

File: locales/zh-TW.json [hunk 4]
```diff
@@ -1,91 +1,91 @@
...
-	"Allow editor to scroll past the last line": "允許編輯器滾動到最後一行",
-	"Bring in web page title when pasting URL on editor": "粘貼網頁鏈接的時候，顯示為網頁標題",
-	"Preview": "預覽器",
-	"Preview Font Size": "預覽器字型大小",
-	"Preview Font Family": "預覽器字體",
-	"Code block Theme": "代碼塊主題",
-	"Allow preview to scroll past the last line": "允許預覽器滾動到最後一行",
-	"Show line numbers for preview code blocks": "在預覽器中顯示行號",
+	"Allow editor to scroll past the last line": "允許編輯器捲軸捲動超過最後一行",
+	"Bring in web page title when pasting URL on editor": "在編輯器貼上網址的時候，顯示為網頁標題",
```
Comment (rayou) [inline locales/zh-TW.json]: 在編輯器貼上網址時，自動添加網頁標題

File: locales/zh-TW.json [hunk 5]
```diff
@@ -1,91 +1,91 @@
...
+	"Preview": "預覽頁面",
+	"Preview Font Size": "預覽頁面字型大小",
+	"Preview Font Family": "預覽頁面字體",
+	"Code block Theme": "程式碼區塊主題",
+	"Allow preview to scroll past the last line": "允許預覽頁面捲軸捲動超過最後一行",
+	"Show line numbers for preview code blocks": "在預覽頁面的程式碼區塊中顯示行號",
 	"LaTeX Inline Open Delimiter": "LaTeX 單行開頭分隔符",
```
Comment (rayou) [inline locales/zh-TW.json]: 分隔符 -> 符號

File: locales/zh-TW.json [hunk 6]
```diff
@@ -1,91 +1,91 @@
...
-	"Facebook Group": "Facebook Group",
+	"Facebook Group": "Facebook 社團",
 	"Twitter": "Twitter",
 	"About": "關於",
 	"Boostnote": "Boostnote",
-	"An open source note-taking app made for programmers just like you.": "一款專門為程式員朋友量身打造的開源筆記",
+	"An open source note-taking app made for programmers just like you.": "一款專門為程式設計師朋友量身打造的開源筆記軟體",
 	"Website": "官網",
 	"Development": "開發",
 	" : Development configurations for Boostnote.": " : Boostnote的開發配置",
```
Comment (rayou) [inline locales/zh-TW.json]: 配置 -> 設定

File: locales/zh-TW.json [hunk 7]
```diff
@@ -1,91 +1,91 @@
...
 	"Analytics": "分析",
-	"Boostnote collects anonymous data for the sole purpose of improving the application, and strictly does not collect any personal information such the contents of your notes.": "Boostnote 收集匿名數據只為了提升軟體使用體驗，絕對不收集任何個人信息（包括筆記內容）",
-	"You can see how it works on ": "你可以看看它的源碼是如何運作的  ",
-	"You can choose to enable or disable this option.": "你可以選擇開啟或不開啟這個功能",
-	"Enable analytics to help improve Boostnote": "允許對數據進行分析，幫助我們改進Boostnote",
+	"Boostnote collects anonymous data for the sole purpose of improving the application, and strictly does not collect any personal information such the contents of your notes.": "Boostnote 收集匿名資料單純只為了提升軟體使用體驗，絕對不收集任何個人資料（包括筆記內容）",
+	"You can see how it works on ": "你可以看看它的程式碼是如何運作的  ",
+	"You can choose to enable or disable this option.": "你可以選擇啟用或禁用這項功能",
+	"Enable analytics to help improve Boostnote": "允許數據分析以協助我們改進Boostnote",
 	"Crowdfunding": "眾籌",
```
Comment (rayou) [inline locales/zh-TW.json]: 群眾募資

File: locales/zh-TW.json [hunk 8]
```diff
@@ -143,9 +143,9 @@
 	"You have to save!": "你必須儲存一下！",
```
Comment (rayou) [inline locales/zh-TW.json]: 😆

File: locales/zh-TW.json [hunk 9]
```diff
@@ -143,9 +143,9 @@
...
 	"Russian": "Russian",
 	"Editor Rulers": "Editor Rulers",
-	"Enable": "開啟",
-	"Disable": "關閉",
-	"Sanitization": "代碼處理",
+	"Enable": "啟用",
+	"Disable": "禁用",
+	"Sanitization": "消毒",
```
Comment (rayou) [inline locales/zh-TW.json]: 過濾 HTML 程式碼

**Comment by rayou** [comment]
Hi @ralphchung 

Please have a look at my suggestions, happy to discuss.

Thanks

File: locales/zh-TW.json (hunk 1 above)
Comment (ralphchung) [reply locales/zh-TW.json]: 其實這個我看原本翻譯廢紙簍感覺也蠻酷的，不過不否認垃圾桶比較親民一點XD

File: locales/zh-TW.json (hunk 6 above)
Comment (ralphchung) [reply locales/zh-TW.json]: 好像應該用 ```組態``` ?

Comment (rayou) [reply locales/zh-TW.json]: yep, sounds good.

**Comment by ralphchung** [comment]
Thank you @rayou for reviewing and giving suggestions.

Will do this in a few days. BTW, I prefer ```廢紙簍``` to ```垃圾桶```. What do you think about this?
Another question, should I add a space between Chinese characters and English characters or other symbols? (my answer is yes!)

Thank you very much again!

**Comment by rayou** [comment]
Hi @ralphchung 

I am fine with `廢紙簍`, and yes, adding a space between Chinese and English characters increases readability, please make these changes when you're available.

Thanks

**Comment by ralphchung** [comment]
@rayou 

I've made the changes and slightly modified some other parts of this file. Please have a look again, happy to discuss too!

Thank you very much!

**Comment by rayou** [comment]
@ralphchung LGTM.

**Comment by kazup01** [comment]
Thanks for your contribution @ralphchung 🎉 and your review @rayou 🍻

**Pull request merged**